scripts/gff.py

```diff
@@ -71,6 +80,24 @@ def save_gff(path: str, df: pd.DataFrame) -> None:
     df : pd.DataFrame
         DataFrame object to write, must have 'header' in attrs
     """
+    
+    # --- START FIX 1: De-duplicate header ---
+    # Ensure the header exists and is a list before de-duplicating
+    if 'header' in df.attrs and isinstance(df.attrs['header'], list):
```

Comment:
Can you add a flag for this like `deuplicated_header` that defaults to false and then only set it to true when you want it to do the deduplication?  Alternatively, a seperate `deduplicate_headers` command in the CLI would actually be ideal.  This kind of hidden, undocumented logic can be confusing or problematic.  This may also affect past logic, so it would be much better if this was not enabled by default.